site/content/in-dev/unreleased/external-idp.md

```diff
@@ -0,0 +1,344 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+title: External Identity Providers 
+type: docs
+weight: 550
+---
+
+Apache Polaris supports authentication via external identity providers (IdPs) using OpenID Connect (OIDC) in addition to the internal authentication system. This feature enables flexible identity federation with enterprise IdPs and allows gradual migration or hybrid authentication strategies across realms in Polaris. 
+
+## Authentication Types 
+
+Polaris supports three authentication modes: 
+
+1. `internal` (Default) 
+   - Only Polaris internal authentication is used. 
+2. `external` 
+   - Authenticates using external OIDC providers (via Quarkus OIDC). 
+   - Disables the internal token endpoint (returns HTTP 501). 
+3. `mixed` 
+   - Tries internal authentication first; if it fails, it falls back to OIDC. 
+
+Authentication can be configured globally or per realm in `application.properties`: 
```

Comment:
I noticed that other docs don't reference the file directly, e.g. see this similar excerpt from the metastores doc:
```
Additionally the retries can be configured via `polaris.persistence.relational.jdbc.*` properties please ref [configuration](./configuration.md)
```

Another doc just lists some properties:
```
It is highly recommended to configure Polaris with previously-generated RSA keys. This can be done
by setting the following properties:
```

Reply:
+1, it's better not to mention this file explicitly since configuration can be done through many different ways (env vars, system props, etc.)
```suggestion
Authentication can be configured globally or per realm by setting the following properties: 
```